lib/src/configcat_client.dart

```diff
@@ -146,8 +146,8 @@ class ConfigCatClient {
       final setting = result.settings[key];
       if (setting == null) {
         final err =
-            'Value not found for key $key. Here are the available keys: ${result.settings.keys.join(', ')}';
-        _errorReporter.error(err);
+            'Failed to evaluate setting \'$key\' (the key was not found in config JSON). Returning the `defaultValue` parameter that you specified in your application: \'$defaultValue\'. Available keys: ${result.settings.keys.join(', ')}.';
```

Comment:
Available keys should be surrounded with single quotes like `'Key1', 'Key2', 'Key3'`